python/sglang/srt/server_args.py

```diff
@@ -508,6 +510,10 @@ def print_deprecated_warning(message: str):
                     "trtllm_mha backend does not support speculative decoding yet."
                 )
 
+        if self.enable_prefill_cuda_graph == True:
+            self.disable_radix_cache = True
+            self.attention_backend = "flashinfer"
```

Comment:
nit: maybe add a logger message to tell users we auto do these several configurations